flake.nix

```diff
@@ -45,7 +46,21 @@
                     };
                   };
                   neovim = prev.neovim.override { withNodeJs = true; };
-
+                  # There is a recent bug that prevents cli --check invocation:
+                  # See: https://github.com/LuaLS/lua-language-server/pull/2775
```

Comment:
Shall we file an issue to remove this once that PR ships?

Reply:
https://github.com/cursorless-dev/cursorless/issues/2613